packages/vitest/src/integrations/chai/poll.ts

```diff
@@ -91,7 +91,7 @@ export function createExpectPoll(expect: ExpectStatic): ExpectStatic['poll'] {
               const rejectWithCause = (cause: any) => {
                 reject(
                   copyStackTrace(
-                    new Error(`Matcher did not succeed in ${timeout}ms`, {
+                    new Error('Matcher did not succeed in time.', {
```

Comment:
Is there any reason for removing `${timeout}ms` from the error message?

Reply:
mostly because the new timeout value is now irrelevant - it's set by Vitest and equal to basically random numbers (to the user) - like 3542ms, 2323ms. I don't think they are worth printing out

Reply:
Ah, I didn't realize `expect.element` is doing `processTimeoutOptions`. It looks good to me :+1: 
If I remember correctly, playwright doesn't change the error message when they patch timeout, so they are lying :smile: